src/app/pages/example/example-dialog/example-dialog.component.html

```diff
@@ -0,0 +1,29 @@
+<h1 class="mat-dialog-title">Add Periodic</h1>
+<div class="mat-dialog-content">
+  <form [formGroup]="periodicForm">
+    <mat-form-field appearance="fill">
+      <mat-label>Position</mat-label>
+      <input matInput placeholder="1" required formControlName="position">
```

Comment:
`required` attribute on the HTML side seems unnecessary, since we have validation on Angular Form side.

Reply:
resolved on last commit